brave/brave-ios
# Fix #7173: NFT tab

## Summary of Changes
Wallet will now have a new tab that displays all selected user's visible NFTs.


This pull request fixes #7173

## Submitter Checklist:


## Test Plan:

1. Make sure you dont have any visible NFT to check the NFT empty state (figma has a note claims Brave only support erc721 token which is not correct since we also support solana NFTs. we removed that note.)
2. Add few NFTs from different networks to check if they are listed after they are added. Check if network label is displayed correct for each NFT. 
3. Check if network filter is working correctly 
4. Check NFT details can be opened correctly
5. Check the send button is not broken in NFT details screen
6. Check edit visible asset button is working and after editing visible asset NFT tab is updated correctly
7. Check edit visible asset(NFT) from Portfolio tab will update NFT tab (vice-versa)


## Screenshots:


![simulator_screenshot_63252B2B-6005-4B90-A1D4-2E00225D66BE](https://user-images.githubusercontent.com/1187676/228954709-a44a6341-fbbc-44fd-acf0-08c68d56f1b6.png)|![simulator_screenshot_7C24F78C-DAD8-4669-9441-DC195D84F9AB](https://user-images.githubusercontent.com/1187676/228954775-18cdebf2-316d-467b-afca-9ee686ca772b.png)
--|--

## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
